File: testprojects/master_scala_mainone/pipelines/ScalaDoAnything/code/src/main/scala/io/prophecy/pipelines/scaladoanything/graph/SQLStatement_1.scala
package io.prophecy.pipelines.scaladoanything.graph

import io.prophecy.libs._
import io.prophecy.pipelines.scaladoanything.config.ConfigStore._
import io.prophecy.pipelines.scaladoanything.config.Context
import io.prophecy.pipelines.scaladoanything.udfs.UDFs._
import io.prophecy.pipelines.scaladoanything.udfs._
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object SQLStatement_1 {

  def apply(context: Context, in0: DataFrame): (DataFrame, DataFrame) = {
    in0.createOrReplaceTempView("in0")
    (context.spark.sql(
       "select * from in0 where in0.`c  - int` > 0 and in0.`c- short` > 1"
     ),
     context.spark.sql(
       "select * from in0 where in0.`c  - int` != '$c_int' and in0.`c-string` not like '$c_sql_pattern'"
     )
    )
  }

}

File: testprojects/master_python_mainone/pipelines/PythonDoAnything/code/pythondoanything/graph/SQLStatement_1.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from prophecy.libs import typed_lit
from pythondoanything.config.ConfigStore import *
from pythondoanything.udfs.UDFs import *

def SQLStatement_1(spark: SparkSession, in0: DataFrame) -> (DataFrame, DataFrame):
    in0.createOrReplaceTempView("in0")
    df1 = spark.sql("select * from in0 where in0.`c  - int` > 0 and in0.`c- short` > 1")
    df2 = spark.sql("select * from in0 where in0.`c  - int` != '$c_int' and in0.`c-string` not like '$c_sql_pattern'")

    return df1, df2
